paddle/ir/core/ir_parser.h

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) 2023 PaddlePaddle Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+#include "paddle/fluid/ir/dialect/pd_type.h"
+#include "paddle/fluid/ir/interface/op_yaml_info.h"
+#include "paddle/ir/core/ir_context.h"
+#include "paddle/ir/core/lexer.h"
+#include "paddle/ir/core/operation.h"
+#include "paddle/ir/core/program.h"
+
+using OpResultMap = std::map<string, ir::OpResult>;
+using AttributeMap = std::unordered_map<std::string, ir::Attribute>;
+using OpAttributeInfoMap = std::map<string, string>;
+
+namespace ir {
+class IrParser {
+ public:
+  Lexer* lexer;
+  IrContext* ctx;
+  OpResultMap opresultmap;
+  Builder* builder;
+  Token last_token;
+  Token cur_token;
+  LexSegment lex_seg;
+  std::string cur_op_name;
+
+ public:
+  explicit IrParser(std::istream& is);
+
+  IrParser(IrContext* ctx, std::istream& is);
+
+  ~IrParser() = default;
+
+  void error();
+
+  Token getToken();
```

Comment:
关于成员函数的命名风格，采用驼峰风格，第一个字母要大写，这里应该改为
```suggestion
  Token GetToken();
```
其他成员函数也需要改下命名风格

Reply:
已修改